jekyll/_cci2/language-android.md

```diff
@@ -102,8 +118,6 @@ Then `./gradlew lint test` runs the unit tests, and runs the built in linting to
 
 We then upload the build reports as job artifacts, and we upload the test metadata (XML) for CircleCI to process.
 
-Nice! You just set up CircleCI for an Android app.
```

Comment:
This seems a lot more informal than most of our documentation.

Reply:
yeah, I think all the language guides have this line :)

Reply:
I think I added this, too. ;)